app/views/posts/index.html.erb

```diff
@@ -1 +1,25 @@
-<h1>This is the first post</h1>
\ No newline at end of file
+<p> This is the list of posts</p>
+
+<span><img src="<%= @user.photo %>" alt="User Photo"></span>
+<span><%= @user.name %></span>
+
+<% @posts.each_with_index do |post, index| %>
+<p>Post #<%= index+1 %></p>
+  <p>
+  <%= post.title %>
+</p>
+
+<p>
+<%= post.text %>
+</p>
+
+<ul>
+  <% post.return_recent_comments.each do |comment| %>
+  <li> <%= comment.text %></li>
+ <% end %> 
+  </ul>
+
+<a href="/users/<%= @user.id %>/posts/<%= post.id %>"> See more</a>
+<% end %>
```

Comment:
| Your Design | Given Design |
| --- | --- |
| ![image](https://github.com/Cabraham1/Blog/assets/93039359/38cb1741-db7a-4643-bd43-3cec9c3f151c) | ![image](https://github.com/Cabraham1/Blog/assets/93039359/00da265d-a847-4147-b34c-32a1c475238d) |

- [ ] Great work on adding the image 👍 but I guess it would be great if you follow the wireframe as suggested by the previous reviewer that your image should be of the same size as the details are which you can see in the wireframe 😊

- [ ] As suggested by the previous reviewer make all your pages look like the given wireframe 👍 so, It would be nice if you organize your user details because if you compare your design with the given design then you would find a lot of things such s padding, margins, spaces, etc which are not similar 🤔 Kindly make these changes to make your project follow the given design 👍